publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/command/ProviderCommands.java

```diff
@@ -14,60 +14,66 @@
  * limitations under the License.
  */
 package com.pavlovmedia.oss.jaxrs.publisher.command;
+ 
 
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Properties;
-import org.apache.felix.scr.annotations.Property;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.Service;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.log.Logger;
+import org.osgi.service.log.LoggerFactory;
+import org.osgi.service.metatype.annotations.Designate;
 
 import com.pavlovmedia.oss.jaxrs.publisher.api.Publisher;
+import com.pavlovmedia.oss.jaxrs.publisher.impl.config.ProviderCommandConfig;
 
 /**
  * Commands to facilitate debugging of the JAX-RS system.
  * 
  * @author Shawn Dempsay {@literal <sdempsay@pavlovmedia.com>}
  *
  */
-@Component
-@Service(ProviderCommands.class)
-@Properties({ 
-    @Property(name = "osgi.command.scope", value = "jax"),
-    @Property(name = "osgi.command.function", value = { "getEndpoints", "getFeatures", "getProviders" }),
-    @Property(name = Publisher.SCAN_IGNORE, value = "true")
-})
+@Component(
+    property= {
+        Publisher.SCAN_IGNORE + "=true"
+    }) 
+@Designate(ocd = ProviderCommandConfig.class)
 public class ProviderCommands {
     @Reference
     Publisher publisher;
     
+    @Reference(service = LoggerFactory.class)
+    Logger logger;
```

Comment:
These are gogo shell commands, they are supposed to output to the system console because they are used interactively, they should not be putting things in the logger

Reply:
You're right. I will correct it.